examples/fishjam-chat/app.json

```
@@ -55,7 +55,7 @@
           }
         }
       ],
-      ["./plugin/build/development-pods-plugin.js"] // Only relevant for local development of packages/ios-client
+      ["./plugin/build/with-custom-development-paths-plugin.js"] // Only relevant for local development of packages/ios-client and packages/android-client
```

WDYT about changing name to better explain what it does. And move comments to plugin itself, that this is plugin required only for example app?
```suggestion
      ["./plugin/build/use-local-paths-for-native-packages.js"] 
```